metadoc-tests/src/test/scala/metadoc/tests/JsonSuite.scala

```diff
@@ -52,38 +52,9 @@ class JsonSuite extends FunSuite with DiffAssertions with BeforeAndAfterAll {
     assertNoDiff(
       index,
       """
-        |symbol: "_root_.com.bar.Main."
```

Comment:
Is it not possible to keep the protobuf text format? The new single-line format makes it difficult to read diffs

Reply:
@thesamet It looks like upgrading to ScalaPB 0.7.0-rc7 (https://github.com/scalameta/scalameta/commit/5a7ab3f1c0a6ec6b8a1f2eeee6e43cee7bf883cf) has changed how the generated classes are prettyprinted. Could you comment on this?

Reply:
Yes, this change of behavior has been asked many times by users. There are instructions on how to get the original behavior of toString here: https://github.com/scalapb/ScalaPB/issues/235#issuecomment-350556157

Alternatively, you can call `toProtoString` instead of `toString`

Reply:
@thesamet Thank you!

@olafurpg Addressed your feedback.